core/src/components/datetime/test/sub-pixel-width/index.html

```diff
@@ -42,6 +42,25 @@ <h2>Modal</h2>
       </ion-modal>
     </ion-content>
   </ion-app>
+
+  <script>
+    const openModal = document.getElementById('open-modal');
+
+    const observer = new MutationObserver(mutationRecords => {
+      if (mutationRecords[0].type === 'characterData') {
+        document.dispatchEvent(new CustomEvent('datetimeCalendarMonthSpecTestEvent'));
+      }
+    });
+
+    openModal.addEventListener('click', () => {
+      observer.observe(document.querySelector('ion-datetime').shadowRoot.querySelector('.calendar-month-year'), {
+        childList: true,
```

Comment:
Is `childList` needed here if we're just checking text content? I do remember there being some cross-browser jank with observing that, so I'm not actually sure 🤔

Reply:
Good catch, that piece wasn't required for listening to slot changes. Removed!